api/client/cli.go

```diff
@@ -123,6 +123,18 @@ func (cli *DockerCli) ImagesFormat() string {
 	return cli.configFile.ImagesFormat
 }
 
+// NetworksFormat returns the format string specified in the configuration.
+// String contains columns and format specification, for example {{ID}}\t{{Name}}
+func (cli *DockerCli) NetworksFormat() string {
+	return cli.configFile.NetworksFormat
+}
```

Comment:
Do we need to expose all of these directly? I was really surprised to see an `ImageFormat()` method on `DockerCli`.

You can access this already from `dockerCli.ConfigFile().NetworksFormat`